docs/CONTRIBUTING.md

```diff
@@ -1,6 +1,52 @@
-As an open source project we welcome contributions of many forms. Please reach out before starting
-work on any major code changes. This will ensure we avoid duplicating work, or that
-your code can't be merged due to a rapidly changing code base. If you would like support
-for a library that is not listed, [contact support][1] to share a request.
+---
 
-[1]: https://docs.datadoghq.com/help
+<p align="center">
+  <strong>
+    <a href="https://github.com/open-telemetry/community#net-auto-instrumentation">Getting Involved<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://gitter.im/open-telemetry/opentelemetry-dotnet-auto-instr">Getting In Touch<a/>
+  </strong>
+</p>
+
+---
+
+# Contributing Guide
+
+We'd love your help!
+
+## Report a bug or requesting feature
+
+Reporting bugs is an important contribution. Please make sure to include:
+
+* Expected and actual behavior
+* OpenTelemetry, OS, and .NET versions you are using
+* If possible, steps to reproduce
+
+## How to contribute
+
+### Before you start
+
+Please read the
+[contribution guide](https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md)
+and the
+[code of conduct](https://github.com/open-telemetry/community/blob/master/code-of-conduct.md).
+for general practices of the OpenTelemetry project.
+
+Select a good issue from the links below (ordered by difficulty/complexity):
+
+* [Good First Issue](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```

Comment:
I like this approach. Something we should post in community repo so every OTel repos follow this consistently?

Reply:
Good idea. We have this on Collector, will suggest this on the community.